fields/validators.py

```diff
@@ -18,6 +22,14 @@ def validate_address(address: str) -> bool:
     return len(address.strip()) > 0
 
 def validate_birthday(birthday: str) -> bool:
-    """Validate the birthday. Expected format is DD.MM.YYYY"""
-    pattern = r"\d{2}.\d{2}.\d{4}"
-    return re.match(pattern, birthday) is not None
+    """Validate the birthday. Expected format is DD.MM.YYYY."""
+    try:
+        datetime.strptime(birthday.strip(), "%d.%m.%Y")
+        return True
+    except ValueError:
+        return False
+
+# add a red star to indicate that the field is required
+def required_field(value: str) -> str:
```

Comment:
Is this used anywhere? I can't find it.

Reply:
This is a feature that adds a red star to a field to show that it is required. But I wasn't sure if it was necessary or not.